composer.json

```
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "symfony/process": "^4.3"
+        "symfony/process": "^5.0"
```

Excuus for our late response (due to vacation), also thank you for your PR.
We have however some projects still using Symfony 4 so both should be supported;

```suggestion
        "symfony/process": "^4.3|^5.0"
```